packages/components/src/styles/display/display.stories.ts

```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use the `sd-display--size-` class to set the display’s size:
+ * - `sd-display--size-default`
+ * - `sd-display--size-3xl`
+ * - `sd-display--size-4xl`
+ */
+
+export const Size = {
+  render: () => html`
+    <div class="flex flex-col gap-6">
+      <div class="sd-display sd-display--size-default">Default</div>
+      <div class="sd-display sd-display--size-3xl">3xl</div>
+      <div class="sd-display sd-display--size-4xl">4xl</div>
+    </div>
+  `
+};
+
+/**
+ * Use the `sd-display--inverted` class to make a display with inverted colors.
  */
 
 export const Inverted = {
-  parameters: { controls: { exclude: ['sd-display--inverted'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-display--inverted', values: [false, true] }]
-      },
-      constants: { type: 'attribute', name: 'sd-display--inverted', value: true },
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['transparent', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      },
-      args
-    });
+  render: () => {
+    return html`<div class="p-4 bg-primary"><div class="sd-display sd-display--inverted">Inverted</div></div>`;
```

Comment:
Please add all three sizes:

<img width="602" alt="image" src="https://github.com/user-attachments/assets/b6325d8d-9a2f-401a-a495-de45ad8b2f5d">

Reply:
Fixed according to updated version from figma:
<img width="414" alt="image" src="https://github.com/user-attachments/assets/f5c928fb-5a0e-40d9-82f9-0e264c3e4825">